FIXEDBY 456ea864147e8add6bb6cd59dd5c287cfd53f282 b95c9925ada6c5e1cae2531db109a449d9dd80b4
diff --git a/db/migrate/20210527172347_add_solvent_to_samples.rb b/db/migrate/20210527172347_add_solvent_to_samples.rb
index a079ed08..9e145dea 100644
--- a/db/migrate/20210527172347_add_solvent_to_samples.rb
+++ b/db/migrate/20210527172347_add_solvent_to_samples.rb
@@ -6,7 +6,7 @@ class AddSolventToSamples < ActiveRecord::Migration[4.2]
     Sample.where(solvent: [nil, '']).where.not(deprecated_solvent: [nil, '']).find_each do |item|
       solvent = Chemotion::SampleConst.solvents_smiles_options.find { |s| item.deprecated_solvent&.include?(s[:label]) }
       new_sol = solvent ? [{ label: solvent[:value][:external_label], smiles: solvent[:value][:smiles], ratio: '1' }].to_json : nil
-      sample.update_column(solvent: new_sol)
+      item.update_columns(solvent: new_sol)
     end
   end
 end
